libmattermost.c

```diff
@@ -4058,6 +4091,13 @@ const gchar *who, const gchar *message, PurpleMessageFlags flags)
 	const gchar *team_id = mm_get_first_team_id(ma);
 	
 	if (room_id == NULL) {
+
+		if (purple_str_has_suffix(who, MATTERMOST_BOT_LABEL)) {
+			purple_notify_error(ma->pc, "Error", "You cannot send instant message to a BOT", "(However you may be able to interact with it using \"/cmd command\" in a chat)", purple_request_cpar_from_connection(ma->pc));
+			//TODO: remove conversation (segfaults in libpurple 2 if called here) 
+			return 0;
+		}
+

```

Comment:
it would be nicer to not open that conversation at all, but when called from libpurple we are already in (some other way of doing it ?)